store/storetest/user_store.go

```diff
@@ -3372,7 +3372,8 @@ func testUserStoreGetProfilesNotInTeam(t *testing.T, ss store.Store) {
 
 	// Add u2 to team 1
 	store.Must(ss.Team().SaveMember(&model.TeamMember{TeamId: teamId, UserId: u2.Id}, -1))
-	u2.UpdateAt = store.Must(ss.User().UpdateUpdateAt(u2.Id)).(int64)
+	u2UserVar, _ := ss.User().UpdateUpdateAt(u2.Id)
```

Comment:
When we replace a `store.Must` with the new approach we capture the error from the return values and check it with `require.Nil(t, err)`

Reply:
You have other cases later where you replace `store.Must` this applies there too.